### PR TITLE
feat: ImportProfile card which use custom rpc call

### PR DESCRIPTION
**Pull request by stizzle123**
Links: https://app.clickup.com/t/2tdm6xm
Screenshot 2022-08-16 at 12 39 26

### CODE REVIEW

File: src/components/endpoints/ImportProfile.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import useWeb3 from '@/compositions/useWeb3'
+import { ref } from 'vue'
+import Notifications from '@/components/Notification.vue'
+import useNotifications from '@/compositions/useNotifications'
+
+const { notification, clearNotification, hasNotification, setNotification } =
+  useNotifications()
+
+const { isAddress } = useWeb3()
+const controllerAddress = ref('')
```
Comment (olusoladavid) [inline src/components/endpoints/ImportProfile.vue:11]: We should be importing the UP address not the controller address

File: src/components/endpoints/ImportProfile.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  if (!isAddress(controllerAddress.value)) {
+    return setNotification(
+      'Please provide a valid controller address',
+      'danger'
+    )
+  }
+  try {
+    await window.ethereum.request({
```
Comment (olusoladavid) [inline src/components/endpoints/ImportProfile.vue:25]: This call returns the new controller address. We should have a second transaction to give the new controller address permissions on the UP

**Review by olusoladavid — CHANGES_REQUESTED** [review]
I think the import flow here was misunderstood - we should be importing a UP?

**Review by dzbo — CHANGES_REQUESTED** [review]
There is a lack of implementation last point:
> once RPC call is triggered we should  have textarea on the card to add controller key permissions back to the deployed profile

File: src/components/endpoints/ImportProfile.vue (hunk 1 above)
Comment (stizzle123) [reply src/components/endpoints/ImportProfile.vue:11]: Thanks for the correction @olusoladavid

**Pull request closed without merging**